.github/pull_request_template.md

```diff
@@ -1,6 +1,6 @@
 ### Summary of Changes
 _Provide a brief summary of changes_
-Closes issue #_
```

Comment:
Because this is so quick, I didn't want to create another full review for it. Needed to remove "issue" word and wanted to keep it a full sentence.